src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -139,51 +122,58 @@ void move() {
     @Test
     @EnabledIfSystemProperty(named = "test.storage.file.huge", matches = "true|on")
     void saveAndLoadHugeFiles() throws Exception {
```

Comment:
@g4s8 this test has no assertions... Maybe we can at least check that storage contains file with given name?